lib/api.js

```diff
@@ -515,6 +521,62 @@ api.declare({
   res.reply(prices);
 });
 
+api.declare({
+  method: 'delete',
+  route: '/credentials',
+  name: 'claimCredentials',
+  title: 'Request worker credentials',
+  stability: API.stability.experimental,
+  input: 'claim-credentials-request.json#',
+  output: 'claim-credentials-response.json#',
+  description: [
+    'Yield credentials for the worker running in the given instance.',
+  ].join(' '),
+}, async function(req, res) {
+  const sIdentDoc = new Buffer(req.body.identityDocument, 'base64').toString();
+  const signature = new Buffer(req.body.signature, 'base64').toString();
+
+  const E = () => {
```

Comment:
I think this name is a little confusing.  `reportError` would be much more clear.  I would like to log these errors with reasons in our logs, so that we can see why any particular request is failing.  Could we have this take an object and message for use in a `log.warn(obj, msg)` call?